examples/airflow-cluster-policy/whirl.setup.d/policies/dag_policy.py

```diff
@@ -0,0 +1,10 @@
+from airflow.models import DAG
+from airflow.exceptions import AirflowClusterPolicyViolation
+
+def dag_policy(dag: DAG):
+    """Ensure that DAG has at least one tag"""
+    if not dag.tags:
+        raise AirflowClusterPolicyViolation(
+            f"DAG {dag.dag_id} has no tags. At least one tag required. File path: {dag.fileloc}"
+        )
+
```

Comment:
Would it make sense to also add an example where we actively add config to dags with a policy? Perhaps something like: 
```python
def task_assign_default_owner(task: BaseOperator):
    if not task.owner or task.owner.lower():
        task.owner = "baboon"
```

Reply:
Created new issue for this